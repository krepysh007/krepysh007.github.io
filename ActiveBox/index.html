<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
	<link rel="stylesheet" href="css/libs.min.css">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<div id="btn_top">
		<a href="#header"><i class="fa fa-angle-double-up"></i></a>
	</div>
	<header class="header" id="header">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<div class="top_header">
						<div class="logo"><a href="#"><img src="img/logo.png" alt="logo"></a></div>
						<nav class="menu" id="menu">
							<div class="btn_menu">
								<i class="fa fa-bars"></i>
							</div>
							<ul>
								<li><a href="#features">Features</a></li>
								<li><a href="#work">Works</a></li>
								<li><a href="#team">Our team</a></li>
								<li><a href="#testimonials">Testimonials</a></li>
								<li><a href="#download">Download</a></li>
							</ul>
						</nav>
					</div>
					<div class="header_text">Your Favorite One Page
						Multi Purpose Template</div>
						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent commodo cursus magna
							vel scelerisque nisl consectetur et.</p>
							<button class="find_more"><a href="#">Find out more</a></button>


						</div>
					</div>
				</div>
			</header>
			<section class="features" id="features">
				<div class="container">
					<div class="row">
						<div class="feature_item">
							<div class="col-md-4 col-sm-6 col-xs-12">
								<div class="item">
									<div class="item_img"></div>
									<h3>Easily Customised</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcor per nulla non metus aucto</p>
								</div>
							</div>
							<div class="col-md-4 col-sm-6 col-xs-12" >
								<div class="item">
									<div class="item_img"></div>
									<h3>Responsive ready</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus aucto</p>
								</div>
							</div>
							<div class="col-md-4 col-sm-6 col-xs-12">
								<div class="item">
									<div class="item_img"></div>
									<h3>Modern Design</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus aucto</p>
								</div>
							</div>
							<div class="col-md-4 col-sm-6 col-xs-12">
								<div class="item">
									<div class="item_img"></div>
									<h3>Clean Code</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus aucto</p>
								</div>
							</div>
							<div class="col-md-4 col-sm-6 col-xs-12">
								<div class="item">
									<div class="item_img"></div>
									<h3>Ready to Ship</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus aucto</p>
								</div>
							</div>
							<div class="col-md-4 col-sm-6 col-xs-12">
								<div class="item">
									<div class="item_img"></div>
									<h3>Download for Free</h3>
									<p>Cras justo odio, dapibus ac facilisis in, egestas eget quam. Nullam quis risus eget urna mollis ornare vel eu leo. Donec ullamcorper nulla non metus aucto</p>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>
			<section class="work">
				<div class="work_list" id="work">
					<div class="work_item">
						<a href="#"><img src="img/work_1.jpg" alt=""></a>
						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_2.jpg" alt=""></a>

						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_3.jpg" alt=""></a>
						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_4.jpg" alt=""></a>
						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_5.jpg" alt=""></a>
						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_6.jpg" alt=""></a>
						<div class="work_focus">
							<div class="project-title">
								<h3>project name</h3>
								<p>User Interface Design</p>
							</div>
						</div>
					</div>
					<div class="work_item">
						<a href="#"><img src="img/work_7.jpg" alt=""></a>				<div class="work_focus">
						<div class="project-title">
							<h3>project name</h3>
							<p>User Interface Design</p>
						</div>
					</div>
				</div>
				<div class="work_item">
					<a href="#"><img src="img/work_8.jpg" alt=""></a>
					<div class="work_focus">
						<div class="project-title">
							<h3>project name</h3>
							<p>User Interface Design</p>
						</div>
					</div>
				</div>
			</div>
		</section>	
		<section class="team" id="team">
			<div class="container">
				<div class="row">
					<div class="col-md-3 col-sm-6 col-xs-12 ">
						<div class="foto"><img src="img/team_1.jpg" alt=""></div>
						<div class="name">Ruth Woods</div>
						<div class="position">Founder, CEO</div>
						<div class="team_text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</div>
						<div class="social">
							<ul>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
							</ul>
						</div>
					</div>
					<div class="col-md-3 col-sm-6 col-xs-12">
						<div class="foto"><img src="img/team_2.jpg" alt=""></div>
						<div class="name">Timothy Reed</div>
						<div class="position">Co-Founder, Developer</div>
						<div class="team_text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</div>
						<div class="social">
							<ul>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
							</ul>
						</div>
					</div>
					<div class="col-md-3 col-sm-6 col-xs-12">
						<div class="foto"><img src="img/team_3.jpg" alt=""></div>
						<div class="name">Victoria Valdez</div>
						<div class="position">UI Designer</div>
						<div class="team_text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</div>
						<div class="social">
							<ul>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
							</ul>
						</div>
					</div>
					<div class="col-md-3 col-sm-6 col-xs-12">
						<div class="foto"><img src="img/team_4.jpg" alt=""></div>
						<div class="name">Beverly Little</div>
						<div class="position">Data Scientist</div>
						<div class="team_text">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed diam eget risus varius blandit sit amet non magna. Nullam quis risus eget urna mollis ornare vel eu leo.</div>
						<div class="social">
							<ul>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
								<li><a href="#"></a></li>
							</ul>
						</div>
					</div>
				</div>
			</div>
		</section>
		<section class="client" id="testimonials">
			<div class="container-fluid">
				<div class="row">
					<div class="col-md-6">
						<div class="client_img"></div>
					</div>
					<div class="col-md-6 ">
						<div class="owl-carousel">
							<div class="item">
								<div class="client_feedback">
									<div class="client_text">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
									<div class="client_name">Susan Sims, Interaction Designer at XYZ</div>
								</div>
							</div>
							<div class="item">
								<div class="client_feedback">
									<div class="client_text">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
									<div class="client_name">Susan Sims, Interaction Designer at XYZ</div>
								</div>
							</div>
							<div class="item">
								<div class="client_feedback">
									<div class="client_text">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
									<div class="client_name">Susan Sims, Interaction Designer at XYZ</div>
								</div>
							</div>
							<div class="item">
								<div class="client_feedback">
									<div class="client_text">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
									<div class="client_name">Susan Sims, Interaction Designer at XYZ</div>
								</div>
							</div>
							<div class="item">
								<div class="client_feedback">
									<div class="client_text">“Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
									<div class="client_name">Susan Sims, Interaction Designer at XYZ</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>
		<section class="download " id="download">
			<div class="container">
				<div class="row">
					<div class="col-md-12 ">
						<div class="download_text">Are You Ready to Start? Download Now For Free!</div>
						<h3>Fusce dapibus, tellus ac cursus commodo</h3>
						<button class="download_here"><a href="#">download here</a></button>
					</div>
				</div>
			</div>
		</section>
		<footer class="footer">
			<div class="container-fluid">
				<div class="row">
					<div class="footer_top">
						<div class="container">
							<div class="col-md-4 first_text">
								<h3>Location</h3>
								<div class="buttom_text">3481 Melrose Place Beverly Hills, CA 90210</div>	
							</div>
							<div class="col-md-4">
								<h3>Share with Love</h3>
								<div class="footer_social">
									<ul>
										<li><a href="#"></a></li>
										<li><a href="#"></a></li>
										<li><a href="#"></a></li>
									</ul>
								</div>
							</div>
							<div class="col-md-4">
								<h3>About ActiveBox</h3>
								<div class="buttom_text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec ullamcorper nulla non metus auctor fringilla.</div>
							</div>
						</div>
					</div>
					<div class="footer_buttom">
						<div class="col-md-12">
							<div class="copy">Copyright © 2015 ActiveBox. All Rights Reserved </div>
							<div class="copy_bottom">Made with Y by Kamal Chaneman</div>
						</div>
					</div>
				</div>
			</div>
		</footer>		


		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
		<script src="js/libs.min.js"></script>
		<script src="js/script.js"></script>
	</body>
	</html>