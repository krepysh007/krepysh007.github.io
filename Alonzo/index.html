<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
	<link rel="stylesheet" href="css/libs.css">
	<link rel="stylesheet" href="css/main.css">

</head>
<body>
	<header>
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<div class="logo "><a href="#"><img src="img/logo.png" alt=""></a></div>
					<div class="contact " ><a href="#contact-us" class="popup-with-form" >contact us</a></div>
					<div class="top_text ">Top-4 <span>models</span></div>
					<div class="top_text_buttom">of card printers</div>
					<div class="top_text_line"></div>
					<div class="top_description">From the leading manufacturer of goods for printing "ZEBRA"</div>
					<div class="buy_link"><a href="#buy_now" class="popup-with-form" >Buy now</a></div><div class="learn_link"><a href="#">Learn more</a></div>
					<div class="printer_img"></div>
				</div>
			</div>
		</div>
	</header>
	<section class="trigers">
		<div class="container">
			<div class="row">
				<div class="col-md-4">
					<div class="icon"><img src="img/trigers_icon1.png" alt=""></div>
					<div class="description">Want to choose a universal and reliable equipment for card printing?</div>
				</div>
				<div class="col-md-4">
					<div class="icon"><img src="img/trigers_icon2.png" alt=""></div>
					<div class="description">Need technical support, maintenance and warranty for printers?</div>
				</div>
				<div class="col-md-4">
					<div class="icon"><img src="img/trigers_icon3.png" alt=""></div>
					<div class="description">Need super-fast printers on plastic cards?</div>
				</div>
				<div class="col-lg-6 col-lg-offset-3 col-md-12">
					<div class="icon_buttom_text">	Choose really reliable equipment from «Zebra» with full support</div>
				</div> 		
			</div>
		</div>
	</section>	
	<section class="product_catalog">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Product catalog</h2>

					<div id="owl-one" class="owl-carousel">

						<div class="owl-item">
							<div class="image_wrapper">
								<div class="main_image"></div>
								<div class="item_image1"></div>
								<div class="item_image2"></div>
								<div class="item_image3"></div>
							</div>

							<div class="text_wrapper">
								<div class="printer_name">ZXP Series 9 Card Printers</div>
								<div class="printer_price">from <span>$125</span></div>
								<div class="printer_description">Engineered to meet your high-security application needs, the ZXP Series 9 ™ lets you produce photo-quality cards quickly and reliably.</div>
								<div class="link_wrap">
									<a class="printer_link">Show more information</a>
								</div>
								<div class="or">or</div>
								<a href="#buy_now" class="buy_now_link popup-with-form" >Buy now</a>
							</div>
						</div>
						<div class="owl-item">
							<div class="image_wrapper">
								<div class="main_image"></div>
								<div class="item_image1"></div>
								<div class="item_image2"></div>
								<div class="item_image3"></div>
							</div>
							
							<div class="text_wrapper">
								<div class="printer_name">ZXP Series 9 Card Printers</div>
								<div class="printer_price">from <span>$125</span></div>
								<div class="printer_description">Engineered to meet your high-security application needs, the ZXP Series 9 ™ lets you produce photo-quality cards quickly and reliably.</div>
								<div class="link_wrap">
									<a class="printer_link">Show more information</a>
								</div>
								<div class="or">or</div>
								<a href="#buy_now" class="buy_now_link popup-with-form" >Buy now</a>
							</div>
						</div>
						<div class="owl-item">
							<div class="image_wrapper">
								<div class="main_image"></div>
								<div class="item_image1"></div>
								<div class="item_image2"></div>
								<div class="item_image3"></div>
							</div>
							
							<div class="text_wrapper">
								<div class="printer_name">ZXP Series 9 Card Printers</div>
								<div class="printer_price">from <span>$125</span></div>
								<div class="printer_description">Engineered to meet your high-security application needs, the ZXP Series 9 ™ lets you produce photo-quality cards quickly and reliably.</div>
								<div class="link_wrap">
									<a class="printer_link">Show more information</a>
								</div>
								<div class="or">or</div>
								<a href="#buy_now" class="buy_now_link popup-with-form" >Buy now</a>
							</div>
						</div>

					</div>
				</div>
			</div>
		</div>
	</section>

	<section class="advantages">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Advantages of the company</h2>
					<div class="col-md-3">
						<div class="advantage_icon"><img src="img/advantages-icon1.png" alt=""></div>
						<h3>Customer Service Support</h3>
						<p>When buying a printer you get detailed advice on choosing a model and its further use, free connection equipment and training work on it.</p>
					</div>
					<div class="col-md-3">
						<div class="advantage_icon"><img src="img/advantages-icon2.png" alt=""></div>
						<h3>Compact size of printers</h3>
						<p>A small case conceals a powerful
							and high-performance design for fast and high-quality printing
							of the materials required by your company - plastic cards</p>
						</div>
						<div class="col-md-3">
							<div class="advantage_icon"><img src="img/advantages-icon3.png" alt=""></div>
							<h3>Stylish ergonomic design</h3>
							<p>The colors and design of each of the models will beautify the office of any company and seamlessly fit into its interior. The printer can be placed
								in any convenient place.</p>
							</div>
							<div class="col-md-3">
								<div class="advantage_icon"><img src="img/advantages-icon4.png" alt=""></div>
								<h3>Optimal cost of cards</h3>
								<p>Printing plastic cards on printers is characterized by a favorable ratio of the price of consumables and the absence of additional costs for the order of work in third-party organizations.</p>
							</div>
						</div>
					</div>
				</div>
		</section>
		<section class="customer">
				<div class="container">
					<div class="row">
						<div class="col-md-12">
							<h2>Customer reviews</h2>
							<div id="owl-two"	class="owl-carousel">
								<div class="owl-item">
									<div class="customer_icon"><img src="img/photo1.jpg" alt="photo1"></div>
									<h3>Rosaline Greene</h3>
									<div class="h3_sub_head">Director of the «LifeStyle Co.»</div>
									<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
									<div class="underline"></div>
								</div>
								<div class="owl-item">
									<div class="customer_icon"><img src="img/photo2.jpg" alt="photo2"></div>
									<h3>Joseph Knight</h3>
									<div class="h3_sub_head">Director of the «Universal Pic.»</div>
									<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
									<div class="underline"></div>
								</div>
								<div class="owl-item">
									<div class="customer_icon"><img src="img/photo3.jpg" alt="photo3"></div>
									<h3>Anne Gregory</h3>
									<div class="h3_sub_head">Director of the «MountCirce»</div>
									<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
									<div class="underline"></div>
								</div>
							</div>
						</div>
					</div>
				</div>
		</section>
			<section class="contact_information">
				<div class="container">
					<row>
						<div class="col-md-12">
							<h2>Contact information</h2>
							<div class="link_map_wrap">
								<span><i class="fa fa-map-marker" aria-hidden="true"></i></span>
								<a href="https://www.google.com.ua/maps/place/795+Park+St,+Brunswick+VIC+3056,+%D0%90%D0%B2%D1%81%D1%82%D1%80%D0%B0%D0%BB%D1%96%D1%8F/@-37.7770737,144.9514953,17z/data=!3m1!4b1!4m5!3m4!1s0x6ad65ccecdc144ed:0x37da864adcc6427f!8m2!3d-37.777078!4d144.953684" class="link_map">Show on Map</a>
							</div>
							<div class="contact_info">
								<ul>
									<span><i class="fa fa-map-marker" aria-hidden="true"></i></span><li>795 South Park Avenue,<br>
										Melbourne, Australia</li>
										<span><i class="fa fa-phone" aria-hidden="true"></i></span><li>8 800 567 890 11<br> 		
											8 800 983 245 88</li>
											<span><i class="fa fa-envelope-o" aria-hidden="true"></i></span><li>info@example.com<br>
												founder@example.com</li>
											</ul>
										</div>
									</div>
								</row>
							</div>
						</section>
					<footer class="footer">
						<div class="container">
							<div class="row">
								<div class="col-md-12">
									<div class="logo_wrap">
										<a href="" class="logo"><img src="img/logo.png" alt="logo"></a>
									</div>
									
									<div class="footer_right">
										<ul>
											<li><i class="fa fa-facebook" aria-hidden="true"></i></li>
											<li><i class="fa fa-twitter" aria-hidden="true"></i></li>
											<li><i class="fa fa-google-plus" aria-hidden="true"></i></li>
											<li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
										</ul>
									</div>
									<div class="copyrigth">© 2017 <a href="#">Alonzo</a> – Free Universal PSD Theme. Design by <a href="https://lpdiamond.com">lpdiamond.com</a></div>
								</div>
							</div>
						</div>
					</footer>
						<form id="contact-us" class="white-popup-block mfp-hide">


							<h4>CONTACT US</h4>
							<ul>
								<li>
									<label for="name">Enter your name</label>
									<input id="name" name="name" type="text" placeholder="Name" required="">
								</li>
								<li>
									<label for="phone">Enter your phone</label>
									<input id="phone" name="phone" type="tel" placeholder="Eg. +447500000000" required="">
								</li>
								<li>
									<label for="email">Enter your email</label>
									<input id="email" name="email" type="email" placeholder="example@domain.com" required="">
								</li>

							</ul>
						</form>	
						<form id="buy_now" class="white-popup-block mfp-hide">


							<h4>BUY NOW</h4>
							<ul>
								<li>
									<label for="name">Enter your name</label>
									<input id="name" name="name" type="text" placeholder="Name" required="">
								</li>
								<li>
									<label for="phone">Enter your phone</label>
									<input id="phone" name="phone" type="tel" placeholder="Eg. +447500000000" required="">
								</li>

							</ul>
						</fieldset>
					</form>			

					<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
					<script src="js/libs.min.js"></script>
					<script src="js/script.js"></script>
					<script src="https://use.fontawesome.com/c12eb79834.js"></script>
				</body>
				</html>