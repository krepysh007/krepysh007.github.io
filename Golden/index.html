<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="libs/font-awesome/css/font-awesome.min.css">
	<link rel="stylesheet" href="libs/bootstrap/bootstrap.min.css">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
<div id="btn_top">
	<a href="#header"><i class="fa fa-angle-double-up"></i></a>
</div>
	<header class="header " id="header" >
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<div class="logo"><a href="#"><img src="img/logo.png" alt=""></a></div>
						<nav class="menu" id="menu">
						<div class="btn_menu">
						<i class="fa fa-bars"></i></div>
					<ul>
						<li><a href="#header">Home</a></li>
						<li><a href="#services">Services</a></li>
						<li><a href="#portfolio">Porfolio</a></li>
						<li><a href="#about">About</a></li>
						<li><a href="#contact">Contact</a></li>
					</ul>
						</nav>
						<div class="home">
							<div class="text1">Welcome To Our Studio!</div>
							<div class="text2">It’s nice to Meet you</div>
							<button class="btn_home">Tell Me More</button>
						</div>
				</div>
			</div>
		</div>
	</header>
	<div class="services" id="services">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>services</h2>
					<h4>Proin iaculis purus consequat sem cure. </h4>
					<div class="items">
						<div class="col-md-4">
						<i class="fa fa-shopping-basket" aria-hidden="true"></i>
							<h3>E-Commerce</h3>
							<div class="text">Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</div>
						</div>
						<div class="col-md-4">
						<i class="fa fa-laptop" aria-hidden="true"></i>
							<h3>Responsive Web</h3>
							<div class="text">Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</div>
						</div>
						<div class="col-md-4">
						<i class="fa fa-lock" aria-hidden="true"></i>
							<h3>Web Security</h3>
							<div class="text">Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
	<div class="portfolio" id="portfolio">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Our portfolio</h2>
					<h4>Proin iaculis purus consequat sem cure.</h4>
				</div>
				<div class="col-md-4">
					<img src="img/port_1.jpg" alt="">
					<h3>Ebony & Ivory</h3>
					<div class="text">Branding</div>
				</div>
				<div class="col-md-4">
					<img src="img/port_2.jpg" alt="">
					<h3>Smart Stationary</h3>
					<div class="text">Print Design</div>
				</div>
				<div class="col-md-4">
					<img src="img/port_3.jpg" alt="">
					<h3>Clever Poster</h3>
					<div class="text">Print Design</div>
				</div>
				<div class="col-md-4">
					<img src="img/port_4.jpg" alt="">
					<h3>Vinyl Record</h3>
					<div class="text">Product Mock-Up</div>
				</div>
				<div class="col-md-4">
					<img src="img/port_5.jpg" alt="">
					<h3>Treehouse Template</h3>
					<div class="text">Web Design</div>
				</div>
				<div class="col-md-4">
					<img src="img/port_6.jpg" alt="">
					<h3>Burned Logo</h3>
					<div class="text">Branding</div>
				</div>
			</div>
		</div>
	</div>
	<div class="about_us" id="about">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>About us</h2>
					<h4>Proin iaculis purus consequat sem cure.</h4>
				</div>
				<div class="items">
					<div class="col-md-6 left">
						<ul>
							<li>
								<h3 class="head">July 2010 </h3>
								<h3 class="sub_head">Our Humble Beginnings</h3>
								<div class="text ">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni 	ssim. Donec porttitora entum suscipit.</div>
								<div class="circle"></div>
							</li>
							<li>
								<h3 class="head">December 2012 </h3>
								<h3 class="sub_head">Enter The Dark Days</h3>
								<div class="text">Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean 
								rhoncus posuere odio in tincidunt.</div>
								<div class="circle"></div>
							</li>
							<li>
								<div class="circle"><h3>Our Story Continues ...</h3></div>
							</li>
						</ul>
					</div>
					<div class="col-md-6 right">
						<ul>
							<li>
								<h3 class="head">January 2011 </h3>
								<h3 class="sub_head">Facing Startup Battles</h3>
								<div class="text">Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni.</div>
								<div class="circle"></div>
							</li>
							<li>
								<h3 class="head">february 2014</h3>
								<h3 class="sub_head">Our Triumph</h3>
								<div class="text">Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean.</div>
								<div class="circle"></div>
							</li>
						</ul>
					</div>
				</div>
			</div>
		</div>
	</div>
	<div class="team">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>our amazing team</h2>
					<h4>Proin iaculis purus consequat sem cure.</h4>
				</div>
				<div class="col-md-4">
					<img src="img/about_1.jpg" alt="">
					<h3>Kimberly Thompson</h3>
					<div class="text">Marketer</div>
					<div class="social">
						<ul>
							<li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a></li>
							</ul>
					</div>
				</div>
				<div class="col-md-4">
					<img src="img/about_2.jpg" alt="">
					<h3>Rico Massimo</h3>
					<div class="text">Coder</div>
					<div class="social">
						<ul>
							<li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i></a></li>
							</ul>
					</div>
				</div>
				<div class="col-md-4">
					<img src="img/about_3.jpg" alt="">
					<h3>Uku Mason</h3>
					<div class="text">Graphic Designer</div>
					<div class="social">
						<ul>
							<li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a></li>
							</ul>
					</div>
				</div>
				<div class="col-md-12">
					<div class="description">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
aenean rhoncus posuere odio in tincidunt proin iaculis.</div>
					<div class="logo_partners">
						<img src="img/logo_1.png" alt="">
						<img src="img/logo_2.png" alt="">
						<img src="img/logo_3.png" alt="">
						<img src="img/logo_4.png" alt="">
					</div>
				</div>
			</div>
		</div>
	</div>
	<div class="contact" id="contact">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
					<h2>Contact us</h2>
					<h4>Proin iaculis purus consequat sem cure.</h4>
				</div>
				<div class="form">
				<form action="">
					<div class="col-md-6">
						<input type="text" placeholder="Your name*">
						<input type="text" placeholder="Your E-mail*">
						<input type="text" placeholder="Subject">
					</div>
					<div class="col-md-6">
						<textarea type="message" placeholder="Your message*"></textarea>
					</div>
					<div class="col-md-12">
						<button class="btn_contact">send message</button>
					</div>
					</form>
				</div>
			</div>
		</div>
	</div>
	<footer class="footer">
		<div class="container">
			<div class="row">
				<div class="col-md-12">
						<div class="social">
						<ul>
							<li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a></li>
							<li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i></a></li>
						</ul>
					</div>
					<div class="copy">© Copyright 2014 FreebiesXpress.com</div>
				</div>
			</div>
		</div>
	</footer>
	
				

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>